app/js/main.js

```diff
@@ -201,6 +211,37 @@ function setupMeteringDemo(subscriptions) {
     location.reload();
   });
 
+  // Determine whether there is a publisher based entitlement.
+  // Showcase should be informed if the publisher unlocks
+  // the page due to using their own meter, managing their
+  // own subscription or because the page is free.
+  // This query params check is for easy testing, replace this
+  // with your own code.
+  switch (getQueryParams().publisherEntitlement) {
+    case ('free', ShowcaseEntitlementEvent.FREE):
+      subscriptions.setShowcaseEntitlement({
+        isUserRegistered: true, // Set this to false if known
+        entitlement: ShowcaseEntitlementEvent.FREE,
```

Comment:
I noticed lines 223 and 237 have comments saying: "Set this to false if known". That seems to imply publishers should set `isUserRegistered` to `true` by default, unless they know otherwise?